rpcs3/Emu/Cell/Modules/cellGcmSys.cpp

```diff
@@ -857,7 +857,7 @@ u64 cellGcmGetVBlankCount()
 {
 	cellGcmSys.trace("cellGcmGetVBlankCount()");
 
-	return rsx::get_current_renderer()->vblank_count;
+	return rsx::get_current_renderer()->vblank_count.load();
```

Comment:
Is this necessary?

Reply:
Not really

Reply:
Why add it then? It may seem minor but picture a PR with dozens of such changes.